git-ftp

```diff
@@ -234,7 +234,7 @@ usage() {
 }
 
 cache_git_submodules() {
-	GIT_SUBMODULES="$(git submodule status -- "$SYNCROOT" 2>/dev/null | grep -v '^-' | awk '{print $2}')"
+	GIT_SUBMODULES="$(git submodule status -- "$SYNCROOT" 2>/dev/null | grep -v '^-' | awk '{$1=$NF=""; print substr($0, 2, length($0) - 2)}')"
```

Comment:
I don't really know `awk` and don't understand how this works, but it seems to do the job pretty well. The only problem is that we are still assigning a string containing newlines to the `GIT_SUBMODULES` variable. It would be better to store each line individually in an array. I think this should work:
```bash
GIT_SUBMODULES=()
while IFS= read -r path; do
	GIT_SUBMODULES+=( "$path" )
done < <( git submodule status -- "$SYNCROOT" 2>/dev/null | grep -v '^-' | awk '{$1=$NF=""; print substr($0, 2, length($0) - 2)}' )
```